### ISSUE TITLE
Device cannot be seen from the new Chromecast with Google TV

### ISSUE THREAD

**Issue by parkerlreed** [AppGround-io/bluetooth-gamepad-support#5]
Related issue from keyboard/mouse https://github.com/AppGround-io/bluetooth-keyboard-and-mouse-support/issues/80

It seems whatever keyboard/mouse and gamepad are doing is not enough to trick the new Chromecast into thinking it's an input device. It never shows up when scanning

EDIT: Using Pixel 3 XL on Android 11

**Comment by parkerlreed** [comment]
I managed to get it paired with a third party application

https://play.google.com/store/apps/details?id=com.waylonhuang.bluetoothpair

I'm not sure how much can be done in the way of the application making it not seem like a phone, so feel free to close this out if needed.

**Comment by AppGround-io** [comment]
Thank you for reporting the problem. It's not possible on Android to change the Bluetooth class as an app, so pairing the devices from the phone is the only option to connect the devices if the remote device is checking for the Bluetooth class. 

Did you try the "Add Device" button in the app to connect to the Chromecast? If the pairing works in "Bluetooth Pair", it should work in the app as well.

**Comment by parkerlreed** [comment]
The Chromecast itself doesn't advertise discovery. Even when on the Bluetooth pairing screen. Tried that from the app and it never showed up. Google only expects you to pair accessories with it, not it to other things.

What I ended up doing was launching th Bluetooth Mouse and Keyboard on the phone, then on the Chromecast running that Bluetooth Pair and then scanning. Connected fine after that. Was able to uninstall Bluetooth Pair afterwards as it's only needed for the initial pairing

Bluetooth Gamepad connected but did not want to work (buttons not doing anything). I'll dig into that some more and file an issue if needed.

Cheers,
-Parker
